packages/@aws-cdk/aws-cloudwatch/lib/metric-types.ts

```diff
@@ -33,48 +44,154 @@ export interface Dimension {
  * Statistic to use over the aggregation period
  */
 export enum Statistic {
-  SAMPLE_COUNT = 'SampleCount',
-  AVERAGE = 'Average',
-  SUM = 'Sum',
-  MINIMUM = 'Minimum',
-  MAXIMUM = 'Maximum',
+  SAMPLE_COUNT = "SampleCount",
+  AVERAGE = "Average",
+  SUM = "Sum",
+  MINIMUM = "Minimum",
+  MAXIMUM = "Maximum"
 }
 
 /**
  * Unit for metric
  */
 export enum Unit {
-  SECONDS = 'Seconds',
-  MICROSECONDS = 'Microseconds',
-  MILLISECONDS = 'Milliseconds',
-  BYTES = 'Bytes',
-  KILOBYTES = 'Kilobytes',
-  MEGABYTES = 'Megabytes',
-  GIGABYTES = 'Gigabytes',
-  TERABYTES = 'Terabytes',
-  BITS = 'Bits',
-  KILOBITS = 'Kilobits',
-  MEGABITS = 'Megabits',
-  GIGABITS = 'Gigabits',
-  TERABITS = 'Terabits',
-  PERCENT = 'Percent',
-  COUNT = 'Count',
-  BYTES_PER_SECOND = 'Bytes/Second',
-  KILOBYTES_PER_SECOND = 'Kilobytes/Second',
-  MEGABYTES_PER_SECOND = 'Megabytes/Second',
-  GIGABYTES_PER_SECOND = 'Gigabytes/Second',
-  TERABYTES_PER_SECOND = 'Terabytes/Second',
-  BITS_PER_SECOND = 'Bits/Second',
-  KILOBITS_PER_SECOND = 'Kilobits/Second',
-  MEGABITS_PER_SECOND = 'Megabits/Second',
-  GIGABITS_PER_SECOND = 'Gigabits/Second',
-  TERABITS_PER_SECOND = 'Terabits/Second',
-  COUNT_PER_SECOND = 'Count/Second',
-  NONE = 'None'
+  SECONDS = "Seconds",
+  MICROSECONDS = "Microseconds",
+  MILLISECONDS = "Milliseconds",
+  BYTES = "Bytes",
+  KILOBYTES = "Kilobytes",
+  MEGABYTES = "Megabytes",
+  GIGABYTES = "Gigabytes",
+  TERABYTES = "Terabytes",
+  BITS = "Bits",
+  KILOBITS = "Kilobits",
+  MEGABITS = "Megabits",
+  GIGABITS = "Gigabits",
+  TERABITS = "Terabits",
+  PERCENT = "Percent",
+  COUNT = "Count",
+  BYTES_PER_SECOND = "Bytes/Second",
+  KILOBYTES_PER_SECOND = "Kilobytes/Second",
+  MEGABYTES_PER_SECOND = "Megabytes/Second",
+  GIGABYTES_PER_SECOND = "Gigabytes/Second",
+  TERABYTES_PER_SECOND = "Terabytes/Second",
+  BITS_PER_SECOND = "Bits/Second",
+  KILOBITS_PER_SECOND = "Kilobits/Second",
+  MEGABITS_PER_SECOND = "Megabits/Second",
+  GIGABITS_PER_SECOND = "Gigabits/Second",
+  TERABITS_PER_SECOND = "Terabits/Second",
+  COUNT_PER_SECOND = "Count/Second",
+  NONE = "None"
+}
+
+/**
+ * Properties of a rendered metric
+ */
+export interface MetricConfig {
+  /**
+   * In case the metric represents a query, the details of the query
+   *
+   * @default unset
```

Comment:
I don't see how "none" is a better phrasing here.

Reply:
The missing `-` is the part I care about. It otherwise isn't uniform with the rest of our documentation. And "the value when you do not set this is 'not set'" feels like a truism that gives the user no additional information.